ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -491,35 +491,36 @@ export const WorkspaceAbout = fp.flow(
                     </div>
 
                     {serverConfigStore.get().config
-                      .enableInitialCreditsExpiration && (
-                      <div style={{ ...styles.infoBox }}>
-                        <div style={styles.infoBoxHeader}>
-                          Workspace Initial Credit Expiration
-                        </div>
-                        <div style={{ fontSize: '0.75rem' }}>
-                          {this.workspaceInitialCreditsExpirationTime}
-                        </div>
-                        <TooltipTrigger
-                          content={initialCreditsExtensionTooltipContent}
-                          disabled={
-                            isWorkspaceCreator &&
-                            profile.eligibleForInitialCreditsExtension
-                          }
-                        >
-                          <LinkButton
+                      .enableInitialCreditsExpiration &&
+                      !workspace.initialCredits.expirationBypassed && (
```

Comment:
what will it look like for BYO user?

Reply:
As long as they were not bypassed, they will still see this box for when their initial credits expire.

[I was discussing this with Sam and Anna over on Slack.](https://pmi-engteam.slack.com/archives/C02MWP2RN5P/p1734473236596729?thread_ts=1734472878.729729&cid=C02MWP2RN5P)